composer.json

```diff
@@ -10,12 +10,12 @@
         "homepage": "https://github.com/S1lentium"
     }],
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=8.0",
```

Comment:
This needs to be ^8.0 so it does not support e.g. php15

Reply:
I don't see this fixed.

_The other thing is that this package is not maintained so this won't get merged anytime soon I believe_

Reply:
why do you say this package is not maintained @S1lentium pushed some changes to this package in june?